### PR TITLE
ACQ - Adding comments to metadb invoice_lines_fund_distributions

### PR DESCRIPTION
**Pull request by sbeltaine**
```
All queries:
- [ ] Pull request is based on a new branch (not main)
- [ ] Pull request scope is not overly broad
- [ ] Release notes have been added or updated in CHANGES.md
- [ ] Query runs without errors
- [ ] Query output is correct
- [ ] Query logic is clear and well documented
- [ ] Query is readable and properly indented with spaces (not tabs)
- [ ] Table and column names are in all-lowercase
- [ ] Quotation marks are used only where necessary

Report queries:
- [ ] Query has complete user documentation
    - [ ] Purpose of report
    - [ ] Sample output
    - [ ] Query instructions

Derived tables:
- [ ] Query begins with user documentation in comment lines
- [ ] File name is listed in `runlist.txt` after dependencies
- [ ] All columns have indexes
- [ ] Table is vacuumed and analyzed
```

### CODE REVIEW

**Comment by sbeltaine** [comment]
Covers Issue #709 
Also updated runlist.txt and CHANGES.md

**Review by VandanaShah-Cornell — DISMISSED** [review]
Everything is fine. 

All queries:
- [x] Pull request is based on a new branch (not main)
- [x] Pull request scope is not overly broad
- [x] Release notes have been added or updated in CHANGES.md
- [x] Query runs without errors
- [x] Query output is correct
- [x] Query logic is clear and well documented
- [x] Query is readable and properly indented with spaces (not tabs)
- [x] Table and column names are in all-lowercase
- [x] Quotation marks are used only where necessary
 
 
Derived tables:
- [x] Query begins with user documentation in comment lines
- [x] File name is listed in `runlist.txt` after dependencies
- [x] All columns have indexes
- [x] Table is vacuumed and analyzed

**Review by trapido — DISMISSED** [review]
Looks great!
All queries:
- [x] Pull request is based on a new branch (not main)
- [x] Pull request scope is not overly broad
- [x] Release notes have been added or updated in CHANGES.md
- [x] Query runs without errors
- [x] Query output is correct
- [x] Query logic is clear and well documented
- [x] Query is readable and properly indented with spaces (not tabs)
- [x] Table and column names are in all-lowercase
- [x] Quotation marks are used only where necessary

Report queries:
- [ ] Query has complete user documentation
    - [ ] Purpose of report
    - [ ] Sample output
    - [ ] Query instructions

Derived tables:
- [x] Query begins with user documentation in comment lines
- [x] File name is listed in `runlist.txt` after dependencies
- [x] All columns have indexes
- [x] Table is vacuumed and analyzed

File: sql_metadb/derived_tables/runlist.txt [hunk 1]
```diff
@@ -39,6 +39,7 @@ instance_tags.sql
 invoice_adjustments_in_addition_to.sql
 invoice_adjustments_ext.sql
 invoice_lines_adjustments.sql
+invoice_lines_fund_distributions.sql
```
Comment (nassibnassar) [inline sql_metadb/derived_tables/runlist.txt:42]: @sbeltaine Here again, invoice_lines_fund_distributions.sql is already present in the runlist.

**Comment by nassibnassar** [comment]
merged in 3c32a30

**Pull request closed without merging**